ras_party/controllers/pending_survey_controller.py

```diff
@@ -73,6 +74,28 @@ def get_users_enrolled_and_pending_survey_against_business_and_survey(business_i
     return total_users
 
 
+@with_db_session
+def pending_survey_deletion(batch_no, session):
+    """
+    Delete pending survey record against batch number
+    :param batch_no: batch number
+    :type batch_no: uuid
```

Comment:
Is the batch no a UUID object or a string?

Reply:
str will change